server/boutique-hotel-technikum/src/main/java/com/example/boutiquehoteltechnikum/dtos/GuestDto.java

```diff
@@ -0,0 +1,24 @@
+package com.example.boutiquehoteltechnikum.dtos;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+@Value
+@Builder
+@Jacksonized
+public class GuestDto {
+
+    @JsonProperty("guestId")
+    private int guestId;
+
+    @JsonProperty("firstName")
+    private String firstName;
```

Comment:
Please add the same additional annotations for validation as in the entities to assure valid dtos are valid entities.